### DISCUSSION TITLE
Item description on the production order screen

### DISCUSSION THREAD

**Discussion by salvastp** [frePPLe/frepple#623] [Q&A]
Hello,

It seems that the item description is only displayed if it is specified in the operation (FrePPLe 8.2.0), whereas the item code is always shown.

I have conducted a test by removing the items from the operations, and the description is not displayed, but its code is.

Best regards,

Thank you.

![image](https://github.com/frePPLe/frepple/assets/129965662/b631ad39-26b1-40b4-b0de-21075ec343df)

**Comment by jdetaeye** [comment]
Well spotted.    I just committed the trivial correction https://github.com/frePPLe/frepple/commit/f8218d0378abf598f5ac3965a2bb487b85c855a3
